quickevent/app/quickevent/plugins/Event/src/services/oresultsclient.h

```diff
@@ -0,0 +1,53 @@
+#ifndef ORESULTSCLIENT_H
+#define ORESULTSCLIENT_H
+
+#pragma once
+
+#include "service.h"
+
+#include <QNetworkAccessManager>
```

Comment:
Must be include here ? Cannot be used like QTimer ?
in header only : ``class QNetworkAccessManager;``

Reply:
It makes no difference. As i see it, `includes` and `class declarations` are used interchangeably in the project. If one is the preferred way, i can change it. It is up to @fvacek

Reply:
I always prefer forward declaration in header file, because it saves compile time.